extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -434,6 +434,49 @@ public function getCuratedContentQuality() {
 		}
 	}
 
+	public function setData( ) {
+		global $wgCityId, $wgEnableCuratedContentUnauthorizedSave;
+		$status = false;
+
+		$this->response->setFormat( WikiaResponse::FORMAT_JSON );
+
+		if ( !empty( $wgEnableCuratedContentUnauthorizedSave ) ) {
+			$helper = new CuratedContentHelper();
+			$data = $this->request->getArray( 'data', [] );
+
+			// strip data
+			foreach( $data as &$section ) {
```

Comment:
@hakubo @rogatty As we only stripping excessive data from an array I don't think it's necessary to create unittest for this functionality - we aren't doing any logic here, so IMHO there's nothing to test.
Validation function is tested itself.